src/crepe/manager/LoopTimerManager.h

```diff
@@ -103,7 +103,7 @@ class LoopTimerManager : public Manager {
 	 *
 	 * \return The fixed delta time, scaled by the current time scale, in seconds.
 	 */
-	duration_t get_scaled_fixed_delta_time() const;
+	float get_scaled_fixed_delta_time() const;
```

Comment:
Dit is inderdaad beter, maar dan moet je dat overal toepassen (dus ook bij de andere 3 getters die momenteel nog `duration_t` returnen)

Reply:
ik had dit aangepast uit feedback van loek dat de chrono waarde returnen beter was. willen we het toch als float waarden returnen?